server/services/offenderService.ts

```diff
@@ -71,8 +81,25 @@ export default function createOffenderService(elite2ClientBuilder): OffenderServ
   const getIncidentLocations = async (token: string, agencyId: AgencyId): Promise<PrisonLocation[]> => {
     try {
       const elite2Client = elite2ClientBuilder(token)
+      const incidentLocations = await elite2Client.getLocations(agencyId)
+
+      const primaryLocations = incidentLocations.filter(
+        loc =>
+          loc.userDescription.toUpperCase() === "PRISONER'S CELL" || loc.userDescription.toUpperCase() === 'OTHER CELL'
```

Comment:
might be a bit cleaner to specify an array of locations considered 'primary'

so this would become something like:

```
const primaryLocationOptions = ["PRISONER'S CELL", 'OTHER CELL']
const primaryLocations = incidentLocations.filter(
  loc => primaryLocationOptions.includes(loc.userDescription.toUpperCase()))
const remainingLocations = incidentLocations.filter(
  loc => !primaryLocationOptions.includes(loc.userDescription.toUpperCase())
)
.sort(compare)

Reply:
You could use [localeCompare](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeCompare) instead of the compare method:
```
.sort((a,b) => a.userDescription.localeCompare(b.userDescription, 'en', {sensitivity: 'base'})
```

Reply:
Or instead of pulling out the separate lists, could do the following:

```
const compare = (a, b): number => {
  if (a.userDescription.toUpperCase() === "PRISONER'S CELL") return -1;
  if (a.userDescription.toUpperCase() === "OTHER CELL") {
    return b.userDescription.toUpperCase() === "PRISONER'S CELL" ? 1 : -1;
  }
  return a.userDescription.localeCompare(b.userDescription, 'en', {sensitivity: 'base'})
}
return incidentLocations.sort(compare)
```

Reply:
used the locale compare.